semanticdb/integration/src/main/scala/example/Flags.scala

```diff
@@ -0,0 +1,17 @@
+package flags
+
+package object p {
```

Comment:
Nice! Golf coding, how many modifiers can you fit in the smaller number of definitions? 😄

Reply:
🤣